spyder/plugins/ipythonconsole/widgets/shell.py

```diff
@@ -1046,11 +1078,31 @@ def _banner_default(self):
 
     def _kernel_restarted_message(self, died=True):
         msg = _("Kernel died, restarting") if died else _("Kernel restarting")
+
+        if died and self.kernel_manager is None:
+            # The kernel might never restart, show position of fault file
+            msg += (
+                "\nCrash file located at: "
+                + self.kernel_handler.fault_filename()
+            )
+
         self.sig_kernel_restarted_message.emit(msg)
 
     def _handle_kernel_restarted(self, *args, **kwargs):
+        """The kernel restarted."""
         super(ShellWidget, self)._handle_kernel_restarted(*args, **kwargs)
```

Comment:
```suggestion
        super()._handle_kernel_restarted(*args, **kwargs)
```

Remove old `super` usage.